src/components/Cards/card-data/full-node-cards.js

```diff
@@ -0,0 +1,46 @@
+const fullNodeCards = [
+  {
+    href: '/full-node/run-a-full-terra-node',
+    lightSvg: '/img/icons/node.svg',
+    darkSvg: '/img/icons/dark/node.svg',
+    header: 'Run a full node',
+    summary: 'Walk through the configuration and management of a full node on Terra.'
+  },
+  {
+    href: '/full-node/run-a-full-terra-node/join-a-network.html',
+    lightSvg: '/img/icons/ecosystem02.svg',
+    darkSvg: '/img/icons/dark/ecosystem02.svg',
+    header: 'Join a network',
+    summary: 'Set up a full node on various Terra networks.'
```

Comment:
I think this card should be more specific:

"Join the mainnet, testnet, or make a local private network. "